modules/statistics/php/stats_demographic.class.inc

```diff
@@ -190,17 +195,17 @@ class Stats_Demographic extends \NDB_Form
         }
 
         //SUBPROJECTS
-        $suproject_query = '';
-        // Only search for subprojects if $currentProject is a positive int.
-        if (is_numeric($currentProject) && intval($currentProject) >= 0) {
+        $subprojectQuery = '';
```

Comment:
The camelCase changes in this file represent the actual names of the variables, not an aesthetic choice. The snake_case variable names are incorrect.